chromium_src/chrome/browser/certificate_manager_model.cc

```diff
@@ -133,8 +133,8 @@ void CertificateManagerModel::DidGetCertDBOnUIThread(
     CreationCallback callback) {
   DCHECK_CURRENTLY_ON(BrowserThread::UI);
 
-  std::unique_ptr<CertificateManagerModel> model(
-      new CertificateManagerModel(cert_db, is_user_db_available));
+  auto model =
+      std::make_unique<CertificateManagerModel>(cert_db, is_user_db_available);
```

Comment:
This class needs some refactoring to be able to be called with `make_unique`.

```
../../buildtools/third_party/libc++/trunk/include/memory:2006:32: error: calling a private constructor of class 'CertificateManagerModel'
    return unique_ptr<_Tp>(new _Tp(_VSTD::forward<_Args>(__args)...));
                               ^
../../electron/chromium_src/chrome/browser/certificate_manager_model.cc:137:12: note: in instantiation of function template specialization 'std::make_unique<CertificateManagerModel, net::NSSCertDatabase *&, bool &>' requested here
      std::make_unique<CertificateManagerModel>(cert_db, is_user_db_available);
           ^
../../electron/chromium_src/chrome/browser/certificate_manager_model.h:94:3: note: declared private here
  CertificateManagerModel(net::NSSCertDatabase* nss_cert_database,
  ^
1 error generated.
```

Reply:
@zcbenz, I'll just remove that one for now. I'm guessing that was a build error on Linux? Thanks for catching it. Looks like the class [has changed upstream](https://source.chromium.org/chromium/chromium/src/+/b429f54e6c9be6a420d01e82e07c93a6912cbe9f) and uses `std::make_unique`, but those changes weren't landed in Electron.